apps/organization/repositories/organization.repository.ts

```diff
@@ -53,7 +53,8 @@ export class OrganizationRepository {
           name: createOrgDto.name,
           logoUrl: createOrgDto.logo,
           description: createOrgDto.description,
-          website: createOrgDto.website
+          website: createOrgDto.website,
```

Comment:
Going forward we should destructure the DTO in the repository.

Reply:
Yes sure. Will do that